src/api/firebase.js

```diff
@@ -186,12 +187,14 @@ export async function addItem(listPath, { itemName, daysUntilNextPurchase }) {
 	return newItem;
 }
 
-export async function updateItem() {
-	/**
-	 * TODO: Fill this out so that it uses the correct Firestore function
-	 * to update an existing item. You'll need to figure out what arguments
-	 * this function must accept!
-	 */
+// export async function updateItem(listPath, itemID) {
+export async function updateItem(listPath, itemID) {
+	const listRef = doc(db, listPath, 'items', itemID);
+
+	await updateDoc(listRef, {
+		dateLastPurchased: new Date(),
+		totalPurchases: increment(1),
```

Comment:
If you want to be able to uncheck, you can do something like this:

```suggestion
export async function updateItem(listPath, itemID, isChecked) {
	const listRef = doc(db, listPath, 'items', itemID);

	await updateDoc(listRef, {
		dateLastPurchased: isChecked ? new Date() : null,
		totalPurchases: isChecked ? increment(1) : increment(-1),
```